src/components/ProfilingToolMenu/BaseProfilingToolMenu.tsx

```diff
@@ -158,7 +158,7 @@ function BaseProfilingToolMenu({isProfilingInProgress = false, pathToBeUsed, dis
                     onToggle={onToggleProfiling}
                 />
             </TestToolRow>
-            {!!pathIOS && (
+            {!!filePath && getPlatform() !== CONST.PLATFORM.WEB && (
```

Comment:
The share button is displayed on the desktop, but it should not be.

Here, We can create a new prop to control the display of the share button. This way, we can remove the platform-specific code.

like below:

```diff
- {!!filePath && getPlatform() !== CONST.PLATFORM.WEB && (
+ {!!filePath && showShareButton && (
```

showShareButton default will be false and we can pass showShareButton from the `src/components/ProfilingToolMenu/index.android.tsx` and `src/components/ProfilingToolMenu/index.ios.tsx`